R/get_code.R

```diff
@@ -4,21 +4,27 @@
 #' to extract user code and solution code.
 #'
 #' @seealso \code{\link{check_result}}, \code{\link{test_result}}, and \code{\link{check_code}}
-#' @export
-#' @rdname get_code
 #' @param user,solution,expr An expression or quosure to evaluate.
 #' @param name Name to print if a \code{NULL} expression is provided.
+#'
+#' @describeIn get_code get student code
 #' @inheritParams rlang::eval_tidy
+#'
+#' @export
 get_user_code <- function(user = NULL, env = rlang::caller_env()) {
   get_code(user, "user", env = env)
 }
+
+#' @describeIn get_code get solution code provided by instructor
+#""
```

Comment:
odd line

Reply:
would "instructor's solution code" be better?

Reply:
oh! Didn't see what was going on.

Let's not export these two functions: `get_user_code` and `get_solution_code`

Reply:
I also `@noRd`'ed `get_code`